code/API_definitions/Application_Endpoint_Discovery_API.yaml

```diff
@@ -7,35 +7,38 @@ info:
     # Connect to the Optimal Application Endpoint
     ---
     # Overview
-    The Application Discovery API extends beyond the capabilities of the
+    The Application Endpoint Discovery API extends beyond the capabilities of the
     Simple Edge Discovery API by not only locating the nearest Edge Cloud Zone
     but also directly linking to the application endpoints within those
```

Comment:
```suggestion
    but also also providing direct links to your application endpoints within those
```

Reply:
^ suggested change because the API provides the links rather than performing the action of linking, and to specify that the developer will only  receive links to their applications (the initial text could be interpreted as meaning 'all application endpoints'